auditbeat/module/file_integrity/event_test.go

```diff
@@ -177,7 +176,7 @@ func TestDiffEvents(t *testing.T) {
 }
 
 func TestHashFile(t *testing.T) {
-	f, err := ioutil.TempFile("", "input.txt")
+	f, err := os.CreateTemp("", "input.txt")
```

Comment:
Suggest replacing this stanza with
```
f, err := os.CreateTemp(t.TempDir(), "input.txt")
if err != nil {
    t.Fatal(err)
}
```
and removing the defer.

Similar at L290 and L363.

Reply:
that should be the default behavior, see https://cs.opensource.google/go/go/+/refs/tags/go1.21.3:src/os/tempfile.go;l=31

Reply:
Note that the suggestion is `t.TempDir()`, not `os.TempDir()`. The change will make a temporary dirctory handled by the testing package and make a file in that directory. The original makes the temp file in the system temp dir.

Reply:
you are correct!